backend/src/Squidex/Config/MyIdentityOptions.cs

```diff
@@ -61,6 +61,13 @@ public sealed class MyIdentityOptions
 
     public string[] OidcScopes { get; set; }
 
+    /// <summary>
+    /// <see cref="Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectOptions"/>
+    /// by default contains scopes "openid" and "profile".
+    /// When <see cref="OidcDisableProfileScope"/> is set to true scope "profile" will be removed.
+    /// </summary>
+    public bool OidcDisableProfileScope { get; set; }
```

Comment:
Please add this documentation to the appsettings, because people actually read it there in constrast to the code.

Reply:
@SebastianStehle You mean like that https://github.com/Squidex/squidex/pull/1207/files#diff-98e0e01be532deff1a844c903b8e00e8f2e31b3c9c789942725db8ea4154280bR691

Reply:
Yes, thank you.